boards/v2/arm/mps3/Kconfig.mps3

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_MPS3_AN547
+	bool
+	default y
```

Comment:
(not this file) this board still have BOARD set which is the old v1 name. Additionally if it is removed then it changes to just "mps3" which is not right as the board name that was supplied was "mps3/an547"

Reply:
fixed here and for renesas board.

thanks for catching